docs/sql-guide/functions/function-offset.md

```diff
@@ -0,0 +1,90 @@
+---
+title: OFFSET()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+
+# RTRIM () function
+
+`OFFSET()` function is used in conjuction with the `LIMIT` clause to control the starting point of data retrieval in a SQL query result. It allows skipping a specified number of rows before fetching the desired number of rows.
+
+## Syntax
+
+```
+limit(num_of_rows) Offset(offset_value)
+```
+
+## Arguments
+
+| Argument | Description |
+|---|---|
+| `num_of_rows` | An `integer` value specifying the maximum number of rows to fetch. |
+| `offset_value` | An `integer` value specifying the number of rows to skip. |
+
+
+## Returns
+
+The `OFFSET()` function doesn't have a direct return value. It affects the result set returned by the SQL query by skipping the specified number of rows before fetching the data.
+
+
+## Examples
+
```

Comment:
I'd put the table definition here as a heading 3 as follows:

```
### Colors table

This table is used in subsequent OFFSET() queries.

create table colors
    (_id id, color string);

insert into colors(_id,color)
    values (1,'green')
insert into colors(_id,color)
    values (2,'red')
insert into colors(_id,color)
    values (3,'yellow')
insert into colors(_id,color)
    values (4,'blue')
insert into colors(_id,color)
    values (5,'orange')
```

Ideally this would be created as a reusable file in: `/_include/sql-guide/statement-table-create-colors.md`

Then it can be inserted here plus in /docs/sql-guide/statements/statement-table-create.md under Examples :)

Reply:
I have moved this under `Examples` header.